src/pages/home/home.css

```diff
@@ -0,0 +1,21 @@
+.home-container{
+    display: grid;
+    grid-template-columns: 250px 1fr 300px;
+}
+.main-container{
+    margin: 4%;
+    
```

Comment:
Confusion: Why margin 4%??

Reply:
Is something wrong in giving in %??

Reply:
Read this if  you still have any confusion !! [this](https://www.hongkiat.com/blog/calculate-css-percentage-margins/)